Makefile

```diff
@@ -13,49 +13,90 @@
 # limitations under the License.
 
 PKG=sigs.k8s.io/ibm-powervs-block-csi-driver
-IMAGE?=quay.io/powercloud/ibm-powervs-block-csi-driver
-VERSION=v0.0.1
 GIT_COMMIT?=$(shell git rev-parse HEAD)
 BUILD_DATE?=$(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
-LDFLAGS?="-X ${PKG}/pkg/driver.driverVersion=${VERSION} -X ${PKG}/pkg/driver.gitCommit=${GIT_COMMIT} -X ${PKG}/pkg/driver.buildDate=${BUILD_DATE} -s -w"
+IMAGE?=ibm-powervs-block-csi-driver
+STAGING_REGISTRY ?= gcr.io/$(PROJECT)
+REGISTRY ?= $(STAGING_REGISTRY)
+STAGING_IMAGE ?= $(STAGING_REGISTRY)/$(IMAGE)
+TAG?=$(GIT_COMMIT)
+STAGING_VERSION=v1.0.0
```

Comment:
why is this assigned to `v1.0.0`

Reply:
Version removed and currently it is expected to fetch from GITTAG